pom.xml

```diff
@@ -51,52 +54,31 @@
     <dependencies>
         <dependency>
             <groupId>com.github.docker-java</groupId>
-            <artifactId>docker-java</artifactId>
-            <version>3.0.13</version>
+            <artifactId>docker-java-api</artifactId>
+            <version>3.2.0-rc4</version>
```

Comment:
Question: Can we use `${revision}` instead of hard-coding the version number (again) here?

It would be nice if we could programatically ensure that the version of this plugin matched the version of the docker-java library that it's including.